src/views/tutorials-landing/components/product-section/index.tsx

```diff
@@ -107,4 +181,29 @@ const ProductSection = ({
 	)
 }
 
+const ProductSection = ({
+	certification,
+	className,
+	featuredCollections,
+	featuredUseCases,
+	product,
+}: ProductSectionProps) => {
+	return (
+		<div className={classNames(s.root, className)}>
+			<MobileProductSection
+				certification={certification}
+				product={product}
+				featuredCollections={featuredCollections}
+				featuredUseCases={featuredUseCases}
+			/>
+			<NonMobileProductSection
+				certification={certification}
+				product={product}
+				featuredCollections={featuredCollections}
+				featuredUseCases={featuredUseCases}
+			/>
+		</div>
```

Comment:
> I think there could have been a more elegant way to handle this where we conditionally add the featuredUseCase to the primary array only for Desktop, and only eject that on mobile.

If you're referring to conditionally rendering parts using JavaScript to determine the viewport size, unfortunately that will cause hydration mismatch errors. This is why I've used the `.g-hide-on-*` classes and rendered some things twice.

> I think this is a scenario where it might have been helpful to have a conversation about this approach before diving into it; I presume it will be prohibitively expensive to undo this, so I won't reject and we can address this later, if we decide that's the right thing to do.

I wish this project had allowed more time for these discussions. 💟